examples/MigrateUsersToPro.ps1

```diff
@@ -0,0 +1,184 @@
+﻿<#
+.Synopsis
+    Migrates users to Pro Power BI licenses.
```

Comment:
Should there be a script for migrating users to standard licenses? Is this operation easy to undo?

Reply:
Actually anyone that isn't pro is a standard\free license but there is a specific license type for standard but I will consider anyone not pro a standard user and I removed the one switch